src/components/dev-hub/breadcrumb-schema.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { useSiteMetadata } from '../../hooks/use-site-metadata';
+
+const getBreadcrumbList = (breadcrumb, siteUrl) => {
+    return breadcrumb.map(({ label, target, to }, index) => {
+        const path = target || to;
+
+        return {
+            "@type": 'ListItem',
+            position: index + 1,
+            name: label,
+            item: path === '/' ? siteUrl : siteUrl + path + '/',
```

Comment:
We have a helper function available called `addTrailingSlashIfMissing` in `~utils/add-trailing-slash-if-missing` which will add the last slash if needed
```suggestion
            item: path === '/' ? siteUrl : addTrailingSlashIfMissing(siteUrl + path),
```